net/http/src/THttpServer.cxx

```diff
@@ -1099,12 +1119,12 @@ Bool_t THttpServer::ExecuteWS(std::shared_ptr<THttpCallArg> &arg, Bool_t externa
       process = handler->HandleWS(arg);
    } else if (arg->fFileName == "root.longpoll") {
       // ROOT emulation of websocket with polling requests
-      if ((arg->fQuery == "connect") || (arg->fQuery == "connect_raw")) {
+      if ((arg->fQuery.Index("raw_connect") == 0) || (arg->fQuery.Index("txt_connect") == 0)) {
```

Comment:
Can you use `Compare()` instead of `Index()==0`?

Reply:
I will use TString::BeginsWith